packages/runtime/rollup.config.js

```diff
@@ -0,0 +1,250 @@
+const path = require('path');
+const babel = require('@rollup/plugin-babel').default;
+const copy = require('rollup-plugin-copy');
+const extensions = require('rollup-plugin-extensions');
+const prettier = require('rollup-plugin-prettier');
+const replace = require('@rollup/plugin-replace');
+const { terser } = require('rollup-plugin-terser');
+const typescript = require('@rollup/plugin-typescript');
+const {
+  babelPluginReplaceVersionPlaceholder,
+  createBanner,
+  getBuildDirectories,
+  validateReplacedVersion,
+  PRETTY,
+} = require('../../rollup.utils');
+const { name, version } = require('./package.json');
+
+module.exports = function rollup() {
+  const { ROOT_DIR, PKG_DIR, SOURCE_DIR, OUTPUT_DIR } = getBuildDirectories(
+    name,
+    'react'
+  );
+
+  const banner = createBanner('Microapp React', version);
+
+  // JS modules for bundlers
+  const modules = [
+    {
+      input: `${SOURCE_DIR}/index.tsx`,
+      output: {
+        file: `${OUTPUT_DIR}/index.js`,
+        format: 'esm',
+        sourcemap: !PRETTY,
+        banner,
+      },
+      external: ['react', 'react-dom'],
+      plugins: [
+        extensions({ extensions: ['.ts', '.tsx'] }),
+        babel({
+          babelHelpers: 'bundled',
+          exclude: /node_modules/,
+          presets: [
+            ['@babel/preset-env', { loose: true }],
+            '@babel/preset-react',
+            '@babel/preset-typescript',
+          ],
+          plugins: [
+            'babel-plugin-dev-expression',
+            babelPluginReplaceVersionPlaceholder(),
+          ],
+          extensions: ['.ts', '.tsx'],
+        }),
+        typescript({
+          tsconfig: path.join(__dirname, 'tsconfig.json'),
+          exclude: ['tests'],
+          noEmitOnError: true,
+        }),
+        copy({
+          targets: [{ src: path.join(ROOT_DIR, 'LICENSE.md'), dest: PKG_DIR }],
+          verbose: true,
+        }),
+        validateReplacedVersion(),
+      ].concat(PRETTY ? prettier({ parser: 'babel' }) : []),
+    },
+  ];
+
+  // JS modules for <script type=module>
+  // Note: These are experimental. You may not even get them to work
+  // unless you are using a React build with JS modules like es-react.
+  const webModules = [
+    {
+      input: `${SOURCE_DIR}/index.tsx`,
+      output: {
+        file: `${OUTPUT_DIR}/react.development.js`,
```

Comment:
Same as my comment above. Make sure to rename all files in to match the package name.

```suggestion
        file: `${OUTPUT_DIR}/runtime.development.js`,
```